app/scripts/modules/azure/serverGroup/configure/serverGroupAdvancedSettingsSelector.directive.html

```diff
@@ -49,13 +49,13 @@
   </div>
 </div>
 <div class="form-group">
-  <div class="col-md-5 sm-label-right"><b>Ramdisk Id (optional)</b></div>
```

Comment:
Is this a bug that these fields are even here?  These look like AWS fields that are result of bad/copy paste.